geolib_plus/bro_xml_cpt/bro_utils.py

```diff
@@ -299,11 +299,15 @@ def get_all_data_from_bro(self, root: _Element) -> None:
         # cpt time of result
         for cpt in root.iter(ns + "conePenetrationTest"):
             for loc in cpt.iter(ns5 + "resultTime"):
-                if loc.text.strip() == "":
+                if loc.text is None:
```

Comment:
since this is changed? is a unit test required? or is it already covered?

Reply:
added a test for that with the files that where wrong